VD_Transmit/VD_Transmit.ino

```diff
@@ -0,0 +1,177 @@
+/* CAN-Bus transmit vehicle dynamics node*/
+
+#include <CAN.h>
+#include <millisDelay.h>
+#include "src/can_conv/sbm_network_definition.h"
+
```

Comment:
No GPS or IMU Messages anymore? 

It doesn't have to be one or the other, you can do both with a second timer

Reply:
What do you mean? At the moment, the transmit ino file is only sending out IMU data (messages 0x102 and 0x103). I'm holding off on sending GPS data since I can really only test that when I'm outside.

Regarding the second timer, would that be to accommodate the different frequencies of the IMU & GPS? The IMU is currently set to run at 208 Hz, and the GPS will be set to 10 Hz.

Reply:
I didn't see the imu message which I thought was 0x100. If you're getting the data you need it's all good. 

As for the gps. Yes I was saying a second timer could be set up for the gps since it's a different rate.